README.md

```diff
@@ -228,28 +239,27 @@ not include -DBINDING_BUILD=Python3 in the CMake command.
 ```
 
 However, if you need to debug the python bindings using an IDE debugger it becomes a little more difficult. 
-The problem is that it requires a debug version of the python library, python37_d.lib.  It is possible to
-obtain one and link with it, but a way to better isolate the python extension is to build a special main( )
-as explained here: https://pythonextensionpatterns.readthedocs.io/en/latest/debugging/debug_in_ide.html.
+The problem is that it requires a debug version of the python library, `python37_d.lib`.  It is possible to
+obtain one and link with it, but a way to better isolate the python extension is to build a special `main( )`
+as [explained in debugging Python](https://pythonextensionpatterns.readthedocs.io/en/latest/debugging/debug_in_ide.html).
 
 Be aware that the CMake maintains a cache of build-time arguments and it will ignore some arguments passed
 to CMake if is already in the cache.  So, between runs you need to clear the cache or even better,
-entirely remove the build/ folder.
+entirely remove the `build/` folder (ie. `git clean -xdf`).
```

Comment:
Actually I use `rm -r build`.  
I did not know that `git clean ` existed.  I will have to give it a try.